vizro-ai/docs/pages/tutorials/explore_vizro_ai.md

```diff
@@ -0,0 +1,131 @@
+# Explore Vizro-AI
+This tutorial serves as an exploration of the extended applications offered by Vizro-AI beyond the initial [quickstart guide](../tutorials/quickstart.md).
+
+By the end of this tutorial, you will have gained an understanding of different language options and leveraging Vizro-AI to enhance the formatting your visualizations.
+
+## Let's get started!
+### 1. Install vizro-ai and get ready to run your code
```

Comment:
`vizro_ai`? 
There are three versions in docs right now. `Vizro-AI`, `vizro_ai` and `vizro-ai`. Do we need to drop `vizro-ai` entirely? I see `vizro-ai` in "explore_vizro_ai.md" and two other files

Reply:
Changed everything to `vizro_ai`